README.md

```diff
@@ -32,49 +38,51 @@ Start only the JSON-RPC server for the SDK you want to test. The JSON-RPC server
 
 Install packages with npm
 
-    npm install
+```
+npm install
+```
 
 Run specific test file
 
-    npm run test test/account/test_accountCreateTransaction.js
+```
+npm run test test/account/test_accountCreateTransaction.js
+```
 
 Run all tests
 
-    npm run test
+```
+npm run test
+```
 
 ### Reports
 
-After running `npm run test` the generated HTML and JSON reports can be found in the mochawesome-report folder
+After running `npm run test` the generated HTML and JSON reports can be found in the `mochawesome-report` folder
 
 ### Linting and Formatting
+
 To ensure code quality and consistent styling, you can run ESLint and Prettier on the codebase.
 
 To check for **code issues**, run:
 
-    npm run lint
+```
+npm run lint
+```
 
 To **format** the code run:
 
-    npm run format
-
-
-## Support
-
-If you have a question on how to use the product, please see our
-[support guide](https://github.com/hashgraph/.github/blob/main/SUPPORT.md).
```

Comment:
Why we did remove the support section?

Reply:
Because it links to Hashgraph. We need to create a new document in Hiero. I created an issue: https://github.com/hiero-ledger/hiero/issues/15